ci/requirements_all.txt

```diff
@@ -3,6 +3,7 @@ pytest-cov
 pytest-xdist
 flake8
 sphinx
```

Comment:
hmm, is this our catch all for what people should install to get a dev env? should add seaborn then